### PR TITLE
adding soxspipe to conda

### PR DESCRIPTION
**Pull request by thespacedoctor**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thespacedoctor** [comment]
@conda-forge/help-python :  This package is ready for review. Many thanks.

File: recipes/soxspipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+test:
+  imports:
+    - soxspipe
+    - soxspipe.commonutils
+    - soxspipe.recipes
+  commands:
+    - soxspipe --help
+  requires:
+    - cov-core
+    - nose2
```
Comment (dopplershift) [inline recipes/soxspipe/meta.yaml:48]: You only need these if you're going to run the test suite, which you're not doing here currently.
```suggestion
```

File: recipes/soxspipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - astropy
+    - bottleneck
+    - ccdproc
+    - docopt
+    - fundamentals
+    - matplotlib
```
Comment (dopplershift) [inline recipes/soxspipe/meta.yaml:29]: Unless you need a pyqt-based GUI, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
This is in great shape, just a couple minor tweaks.

File: recipes/soxspipe/meta.yaml (hunk 1 above)
Comment (thespacedoctor) [reply recipes/soxspipe/meta.yaml:48]: ah yes - forgot!

**Pull request merged**